cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/dataset/BusinessMetadataRecord.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.cdap.data2.metadata.dataset;
+
+import co.cask.cdap.proto.Id;
+
+import java.util.Objects;
+import javax.annotation.Nullable;
+
+/**
+ * Represent Busineess Metadata for CDAP.
+ */
+public class BusinessMetadataRecord {
+  private final String targetType;
+  private final Id.NamespacedId  targetId;
+  private final String key;
+  private final String value;
+  private final String schema;
+
+  public BusinessMetadataRecord(String targetType, Id.NamespacedId targetId, String key, String value) {
```

Comment:
I think you missed the comment about ``targetType``. It can be derived from ``targetId``. We shouldn't have to pass it from outside